src/controller_examples/rabbitmq_controller/proof/liveness/helper_invariants/invariants.rs

```diff
@@ -110,6 +112,72 @@ proof fn lemma_server_config_map_create_request_msg_implies_key_in_reconcile_equ
     seq_lemmas::seq_equal_preserved_by_add(key.name, cr_key.name, new_strlit("-server-conf")@);
 }
 
+/// This lemma is used to show that if an action (which transfers the state from s to s_prime) creates a server config map
```

Comment:
Nice! Thanks for the efforts to improve the readability and maintainability of the proof.